src/main/java/org/takes/rs/RsWithoutHeader.java

```diff
@@ -50,13 +51,14 @@ public RsWithoutHeader(final Response res, final CharSequence name) {
             new ResponseOf(
                 () -> {
                     final String prefix = String.format(
-                        "%s:", new EnglishLowerCase(name.toString()).string()
+                        "%s:", new UncheckedText(
+                            new Lowered(name.toString())
+                        ).asString()
                     );
                     return new Filtered<>(
-                        header -> {
-                            return !new EnglishLowerCase(header).string()
-                                .startsWith(prefix);
-                        },
+                        header -> !new UncheckedText(
+                            new Lowered(header)
+                        ).asString().startsWith(prefix),
```

Comment:
@fanifieiev please, leave a puzzle for [it](https://github.com/yegor256/cactoos/issues/1265).